### PR TITLE
use microservices-common to manage the exceptions

### PR DESCRIPTION
**Pull request by MaelAudren**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -87,7 +87,7 @@ licenseReport {
 
 repositories {
 
-    if (project.hasProperty('local')) {
+   if (project.hasProperty('local')) {
```
Comment (tao-xinxiu) [inline build.gradle]: it seems missing a space here.

File: src/integration-test/resources/archives/workflow_0.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tao-xinxiu) [inline src/integration-test/resources/archives/workflow_0.xml]: The file could be renamed more meaningful as "workflow_hello_world" ?

File: build.gradle (hunk 1 above)
Comment (MaelAudren) [reply build.gradle]: Done

File: src/integration-test/resources/archives/workflow_0.xml (hunk 2 above)
Comment (MaelAudren) [reply src/integration-test/resources/archives/workflow_0.xml]: I updated the code to remove this file.

File: src/main/java/org/ow2/proactive/catalog/rest/exceptiohandler/ExceptionHandlerController.java [hunk 3]
```diff
@@ -49,33 +43,16 @@
 @Controller
 @ControllerAdvice
 @Log4j
-public class ExceptionHandlerController {
-
-    @ExceptionHandler(Exception.class)
```
[the remark this answers is not in the document]
Comment (MaelAudren) [reply src/main/java/org/ow2/proactive/catalog/rest/exceptiohandler/ExceptionHandlerController.java]: ExceptionHandlerController extends ExceptionHandlerAdvice which have a general exception handler.

**Pull request merged**